### PR TITLE
Category update to old Poznań maps

### PR DESCRIPTION
**Pull request by Asteliks**
Added the category historicphoto to maps form 2014 and 2016 as there is a new map in town called [PoznaOrthophotomapBiezaca(aerialimage)](https://github.com/osmlab/editor-layer-index/blob/gh-pages/sources/europe/pl/PoznaOrthophotomapBiezaca(aerialimage).geojson)

### CODE REVIEW

**Comment by Asteliks** [comment]
could I ask for a "hacktoberfest-accepted" label for this PR?

**Comment by andrewharvey** [comment]
Looks like the check failure is because

> ERROR:root:Layers 'ortofotomapa_2014_image' not advertised by WMS GetCapabilities request.

**Comment by Asteliks** [comment]
yea something is way off :(
It all went south after GitHub found a collision that needs to be fixed some time before yesterday. 
As for the error the server might be dead? The strange thing is [WMS GetCapabilities query](https://mybinder.org/v2/gh/rbuffat/eli-helper/master?urlpath=apps%2F/create_wms_template_appmode.ipynb) can no longer get the proper link from the GetCapabilities XML.

try using this link in the tool mentioned above:
`http://wms1.geopoz.poznan.pl:6080/arcgis/services/sip/ortofotomapy/MapServer/WMSServer?service=wms&version=1.3.0&request=GetCapabilities`

as you can see this is the updated link fresh from their [site](http://sip.geopoz.pl/sip/uslugi/get_uslugi_lay/). In the "Nazwa warstwy" write
`ortofotomapa_2014_image`

We may have similar issues in the future with layers called:
`ortofotomapa_2016_cir_image`
`ortogpkg:ortofotomapa_biezaca`

Should anyone know what's and should it be on the Geopoz side please comment, I can contact them later.

I think I might need to scrap this PR and find out what has changed, why and what to do with it

**Comment by Asteliks** [comment]
the the new links seem to fix everything. I'll also need to update the one for `ortogpkg:ortofotomapa_biezaca` added in this merge https://github.com/osmlab/editor-layer-index/pull/935/files

As I get it the old link should still be working but that might change

**Comment by Asteliks** [comment]
Ones again ready for merge

File: sources/europe/pl/PoznaOrthophotomap2016(aerialimage).geojson [hunk 1]
```diff
@@ -1,25 +1,31 @@
...
-        },
-        "max_zoom": 23,
-        "category": "historicphoto"
+  "type": "Feature",
+  "properties": {
+    "id": "poznan-ortofotomapa2016",
+    "name": "Poznań: Orthophotomap 2016 (aerial image)",
+    "type": "wms",
+    "url": "http://wms1.geopoz.poznan.pl/arcgis/services/sip/ortofotomapy/MapServer/WMSServer?LAYERS=ortofotomapa_2016_image&STYLES=default&FORMAT=image/jpeg&CRS={proj}&WIDTH={width}&HEIGHT={height}&BBOX={bbox}&VERSION=1.3.0&SERVICE=WMS&REQUEST=GetMap",
+    "license_url": "https://www.geopoz.pl/bip/index.php?t=200&id=2449",
```
Comment (andrewharvey) [inline sources/europe/pl/PoznaOrthophotomap2016(aerialimage).geojson]: This link just seems to take me to the home page? Is there somewhere that lists the licensing?

**Comment by Asteliks** [comment]
it would seem that in the mean time they have not only updated the links to the map layers but also their information webpage. Wow what luck. Well at least I'm here to fix it

The publish date of the new website:
image

**Comment by Asteliks** [comment]
of course the changes needed are to all the maps from them

**Comment by Asteliks** [comment]
The license and privacy url update for `PoznaOrthophotomapBiezaca(aerialimage)` can be found here https://github.com/osmlab/editor-layer-index/pull/957

PS. Can I get the "hacktoberfest-accepted" label for my PR

**Comment by andrewharvey** [comment]
Same comment from me at https://github.com/osmlab/editor-layer-index/pull/957#issuecomment-713275368

Overall, appreciate the PRs here, but I just want to make sure we absolutely can use this source.

**Comment by michalmich** [comment]
Hi, I am the author of OGC services in Geopoz.
The WMS service called ortogpkg: ortofotomapa_biezaca always has the latest orthophotomap for Poznań available.
Servers with wms1 in the name contain archived orhotophotos.
Please add me to the contact michal.michalowski@geopoz.poznan.pl in case of problems with the availability or change of the address of the services. greetings

**Comment by andrewharvey** [comment]
@michalmich , @Asteliks pointed to you in #957, my questions there are at https://github.com/osmlab/editor-layer-index/pull/957#issuecomment-713275368, where I raised some point where there might be a licensing issue. If you had any comments about the license there please chime in.

Aside, personally I'm okay with reviewing CC BY + waiver or public domain, but all these custom licenses mean we need to manually review each with the limited legal resources, so I am outside my comfort zone with all of these PRs with custom licenses.

**Comment by rbuffat** [comment]
@andrewharvey If the license situation is unclear, would it be better to ask for explicit permission from the data owners? 

We should state how the data is used by openstreetmap, what data is derived under what license, and how attribution to the data used is given. Then the data owners can decide if this is ok for them. Is there a template that includes all this information in a short and concise form?

CC @grischard @simonpoole

**Comment by Asteliks** [comment]
@andrewharvey could you e-mail @michalmich as he seems to have GitHub notifications turned off

**Comment by Asteliks** [comment]
Besides the potential law changes I have an appointment with Zawadzki Jarosław. He is the deputy director of GEOPOZ. If all goes well we will get all the license changes tomorrow

**Comment by Asteliks** [comment]
@rbuffat can we rerun the auto check and merge this?

**Pull request closed without merging**